src/App.tsx

```diff
@@ -1,58 +1,112 @@
-import React from 'react';
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
-import './App.css';
+import React, { useState } from 'react';
+import styles from './App.module.scss';
+import Modal from "./components/Modal/Modal";
+import ContactsModalBody from "./components/ContactsModal/Body";
+import ContactsModalFooter from "./components/ContactsModal/Footer";
+import CustomButton from "./components/Button/Button";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <Counter />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <span>
-          <span>Learn </span>
-          <a
-            className="App-link"
-            href="https://reactjs.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            React
-          </a>
-          <span>, </span>
-          <a
-            className="App-link"
-            href="https://redux.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Redux
-          </a>
-          <span>, </span>
-          <a
-            className="App-link"
-            href="https://redux-toolkit.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Redux Toolkit
-          </a>
-          ,<span> and </span>
-          <a
-            className="App-link"
-            href="https://react-redux.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            React Redux
-          </a>
-        </span>
-      </header>
-    </div>
-  );
+    const [showModalA, setShowModalA] = useState(window.location.hash === '#ModalA');
+    const [showModalB, setShowModalB] = useState(window.location.hash === '#ModalB');
+    const [onlyEven, setOnlyEven] = useState(false);
+
+    const openModalAHandler = () => {
+        setShowModalA(true);
+        window.location.hash = '#ModalA';
+    }
+
+    const openModalBHandler = () => {
+        setShowModalB(true);
+        window.location.hash = '#ModalB';
+    }
+
+    const closeModalAHandler = () => {
+        setShowModalA(false)
+        window.location.hash = ''
```

Comment:
Avand react router instalat in aplicatie ai fi trebuit sa folosesti useLocation si navigate in loc de window.

Cum ar fi acest exemplu:
![image](https://user-images.githubusercontent.com/22646067/196372895-8d8372d7-f36c-4566-8590-5f84e03e3b39.png)